be/src/storage/tablet_updates.cpp

```diff
@@ -2971,6 +2981,17 @@ void TabletUpdates::_remove_unused_rowsets() {
     }
 }
 
+Status TabletUpdates::check_and_remove_rowset() {
+    _remove_unused_rowsets(true);
+    if (!_unused_rowsets.empty()) {
+        std::string msg =
+                strings::Substitute("some rowsets of tablet: $0 are still been referenced", _tablet.tablet_id());
+        LOG(WARNING) << "msg";
```

Comment:
```suggestion
        LOG(WARNING) << msg;

Reply:
> 

done, changed in https://github.com/StarRocks/starrocks/pull/15959/commits/a8c877d82a733a59318f35f294a3980c41f1b4ac